includes/scripts.php

```diff
@@ -432,3 +432,35 @@ function give_admin_hide_notice_shortly_js() {
 }
 
 add_action( 'admin_head', 'give_admin_hide_notice_shortly_js' );
+
+/**
+ * Admin js code
+ *
+ * This code helps to hide php version upgrade nag for 24 hour if admin user dismissed notice.
+ *
+ * @since 1.8.9
+ *
+ * @return void
+ */
+function give_admin_hide_outdated_php_notice_shortly_js() {
```

Comment:
@DevinWalker I've made the output of JS only if the notice is displaying, otherwise JS will not be triggered. Also, tested the same with changing PHP versions as well as during the period of dismissal of notice.